app-emulation/fex-rootfs-mesa-asahi/fex-rootfs-mesa-asahi-20240909.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit systemd linux-info
+
+DESCRIPTION="FEX rootfs overlay containing asahi mesa"
+HOMEPAGE="https://github.com/WhatAmISupposedToPutHere/fex-rootfs"
+
+SRC_URI="
+	https://github.com/WhatAmISupposedToPutHere/fex-rootfs/releases/download/mesa-${PV}/fex-mesa.sqfs -> ${P}.sqfs
+	https://github.com/WhatAmISupposedToPutHere/fex-rootfs/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz
+"
+
+S="${WORKDIR}/fex-rootfs-${PV}"
+
+LICENSE="metapackage MIT"
+SLOT="0"
+KEYWORDS="-* ~arm64"
+DEPEND="
+	sys-apps/systemd
```

Comment:
This should be behind the systemd USE flag so that folks don't inadvertently end up pulling systemd in